javascript/new-src/src/js/app.js

```diff
@@ -0,0 +1,497 @@
+/**
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+angular
+  .module('olaApp', ['ngRoute'])
+  .constant('Channels', 512)
```

Comment:
Can you name these to be closer to either the JS or C++ constant names: https://github.com/OpenLightingProject/ola/blob/master/javascript/ola/common/dmx_constants.js or https://github.com/OpenLightingProject/ola/blob/master/include/ola/Constants.h